### ISSUE TITLE
Auto Join doesnt work

### ISSUE THREAD

**Issue by v6r** [Stanley-GF/Shark.Solar#52]
It create tokens but they dont join anymore. Also changed the proxys and invite a few times

**Comment by Bl4ckBl1zZ** [comment]
yes because he is retarded and leaked it. now it doesn't work anymore

**Comment by Stanley-GF** [comment]
No stfu

**Comment by xenos1337** [comment]
proof

**Comment by Bl4ckBl1zZ** [comment]
> No stfu

I have this tool from dort. Since you leaked it, the tokens don't join the servers anymore.

**Comment by v6r** [comment]
> > No stfu
> 
> I have this tool from dort. Since you leaked it, the tokens don't join the servers anymore.
 
what is dorts github/Discord

**Comment by Bl4ckBl1zZ** [comment]
> > > No stfu
> > 
> > 
> > I have this tool from dort. Since you leaked it, the tokens don't join the servers anymore.
> 
> what is dorts github/Discord

You cant fix it, discord has fixed this. the tool is basically useless since it doesn't join the server

**Comment by Locksdsc** [comment]
> what is dorts github/Discord

https://github.com/DortDev but there's no gen now :/

**Comment by Locksdsc** [comment]
> You cant fix it, discord has fixed this. the tool is basically useless since it doesn't join the server

you can still auto join discord has not patched it

**Comment by Bl4ckBl1zZ** [comment]
> > what is dorts github/Discord
> 
> https://github.com/DortDev but there's no gen now :/

I'm pretty sure the gen was never on the github, it was for sale for about 50 dollars. I got it and now since this had been leaked on github, the tools basically doesn't fully work anymore

**Comment by Fweak** [comment]
poorons.

**Comment by Stanley-GF** [comment]
yall stupid or what? the error's comming from the request. If you put the hcaptcha response key in another /register request, it'll work perfectly. The error's comming from the python script, but i'm not a python developer, so i'll not patch this. You can just send the hcaptcha response key to an api in nodejs & make the request in NodeJS

**Comment by Fweak** [comment]
lol\

**Comment by gatesthebill** [comment]
how to fix the auto join, tokens get fqcked when right after creation

**Comment by Locksdsc** [comment]
or you can just change headers.

**Comment by Fweak** [comment]
> > yall stupid or what? the error's comming from the request. If you put the hcaptcha response key in another /register request, it'll work perfectly. The error's comming from the python script, but i'm not a python developer, so i'll not patch this. You can just send the hcaptcha response key to an api in nodejs & make the request in NodeJS
> 
> Nah you're hella stupid @Stanley-GF ....
> Discord patched everything including requests. You can't do shit like joining over requests anymore.
> Guess it was a veryyy great idea releasing this on github.
> Fucking idiot

If a regular user can do it. A bot can as well.
